Frends.Oracle.ExecuteQuery/Frends.Oracle.ExecuteQuery/Frends.Oracle.ExecuteQuery.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Oracle.ManagedDataAccess.Client;
+using System.ComponentModel;
+using Frends.Oracle.ExecuteQuery.Definitions;
+using Newtonsoft.Json.Linq;
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Frends.Oracle.ExecuteQuery.Tests")]
+namespace Frends.Oracle.ExecuteQuery
+{
+    /// <summary>
+    /// Task class
+    /// </summary>
+    public class Oracle
+    {
+        /// <summary>
+        /// Task for performing queries in Oracle database.
+        /// [Documentation](https://tasks.frends.com/tasks#frends-tasks/Frends.Oracle.ExecuteQuery)
+        /// </summary>
+        /// <param name="connection">Properties to establish connection to Oracle databse</param>
+        /// <param name="properties">Properties for the query to be executed</param>
+        /// <param name="options">Task options</param>
+        /// <param name="cancellationToken">CancellationToken is given by Frends UI</param>
+        /// <returns>Object { bool Success, int RowsAffected, List Output }</returns>
```

Comment:
Output list should be opened up to let users know what properties the objects in that list have.

Reply:
The output depends on the Select query. It is a List of generic JObjects that can have any attributes that the Select query is getting from database. When doing some other query (Insert, Update, Delete) the Output is Empty List.